pkg/plugin/comment_service.go

```diff
@@ -0,0 +1,103 @@
+package plugin
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/arquillian/ike-prow-plugins/pkg/scm"
+	"github.com/arquillian/ike-prow-plugins/pkg/utils"
+	"github.com/google/go-github/github"
+	"github.com/sirupsen/logrus"
+)
+
+// IkePluginsTitle is a constant containing "Ike Plugins" title with markdown formatting
+const IkePluginsTitle = "### Ike Plugins"
+
+// CommentService is a struct managing plugin comments
+type CommentService struct {
+	client         *github.Client
+	log            *logrus.Entry
+	issue          RepositoryIssue
+	commentContext CommentContext
+}
+
+// RepositoryIssue holds owner name, repository name and an issue number
+type RepositoryIssue struct {
+	Owner    string
+	RepoName string
+	Number   int
+}
+
+// CommentContext holds a plugin name and a assignee to be mentioned in the comment
+type CommentContext struct {
+	PluginName string
+	Assignee   string
```

Comment:
We might have multiple Assignees in GH (especially reviewers).

Reply:
I think that I asked you this question - who should be mention in the comment - or maybe I didn't. 
But again, for the purpose of this plugin, this seemed to be sufficient - I didn't want to speculate about the other usages. When we want to bring support for the other plugins, then we can think about more assignees. 
in addition, I'm not sure what that would work - the comment is added once when the PR is opened and the "owner" of the PR is the only person that is connected to it

Reply:
`// TODO rethink this naming when plugins will start interacting with issue creators and reviewers`

Reply:
e1bed20